src/Orleans.Core.Abstractions/Placement/PlacementStrategy.cs

```diff
@@ -5,5 +5,10 @@ namespace Orleans.Runtime
     [Serializable]
     public abstract class PlacementStrategy
     {
+        /// <summary>
+        /// Returns a value indicating whether or not this placement strategy requires activations to be registered in
+        /// the grain directory.
+        /// </summary>
+        public virtual bool IsUsingGrainDirectory => true;
```

Comment:
Will this break backward compatibility with 2.0 and 2/1?

Reply:
It will because `PlacementStrategy` is sent from silo-to-silo when we merge the list of supported grains.

Reply:
This isn't serialized, though: it's not backed by a field.

If it was defined as follows, then that would break serialization:

```C#
public virtual bool IsUsingGrainDirectory { get; } = true;
```

Reply:
That's good. Will it work though for cluster in a (temporary) hybrid state with some silos and clients running a version (2.1) without this change and some a version (2.2) with it?

Reply:
I believe so - there's no serialization or behavioral change in this PR until we start adding new placement strategies.